packages/web-app-external/src/index.js

```diff
@@ -15,11 +10,11 @@ const appInfo = {
 const routes = [
   {
     name: 'apps',
-    path: '/:fileId/:appName?',
+    path: '/:filePath*',
     component: App,
     meta: {
       auth: false,
-      title: $gettext('External app')
```

Comment:
Is this nowhere used?

Reply:
No, we overwrite the title in the code to include the file and the app name (I think this should be centralized somehow as well...)

Reply:
We should keep apps in mind that don't handle files, so the filename part should be optional imho :)